platform-docs/docs/basic-concepts/data.md

```diff
@@ -100,7 +100,7 @@ After running this through the parser, we're left with a simple object we can ma
 
 This has dramatic implications for how simple and performant we can make our parser. These explicit boundaries also protect damage to a single block from bleeding into other blocks or tarnishing the entire document. It also allows the system to identify unrecognized blocks before rendering them.
 
-_N.B.:_ The defining aspects of blocks are their semantics and the isolation mechanism they provide: in other words, their identity. On the other hand, where their data is stored is a more liberal aspect. Blocks support more than just static local data (via JSON literals inside the HTML comment or within the block's HTML), and more mechanisms (_e.g._, global blocks or otherwise resorting to storage in complementary `WP_Post` objects) are expected. See [attributes](/docs/reference-guides/block-api/block-attributes.md) for details.
+_N.B.:_ The defining aspects of blocks are their semantics and the isolation mechanism they provide: in other words, their identity. On the other hand, where their data is stored is a more liberal aspect. Blocks support more than just static local data (via JSON literals inside the HTML comment or within the block's HTML), and more mechanisms (_e.g._, global blocks or otherwise resorting to storage in complementary `WP_Post` objects) are expected. See [attributes](https://github.com/WordPress/gutenberg/tree/HEAD/docs/reference-guides/block-api/block-attributes.md) for details.
```

Comment:
I think maybe we should remove the link for now, until we have a dedicated reference for attributes (to avoid confusing them with WP jargon)

Reply:
I see. I deleted the entire sentence including the link, is this correct?